styles/FeaturedWork.module.css

```diff
@@ -0,0 +1,84 @@
+.featured-work {
+  margin-top: 2%;
+  margin-bottom: 2%;
+  display: flex;
+  border-bottom: #000000 solid 1px;
+  border-top: #000000 solid 1px;
+}
+
+.featured-work__left-container {
+  width: 33%;
+  margin-right: 2%;
+  margin-bottom: 2%;
+  margin-top: 2%;
```

Comment:
Same here, a single margin can be used instead of multiple margins.

Reply:
Thanks, it is fixed